test/integration/krane_deploy_test.rb

```diff
@@ -1521,30 +1521,30 @@ def test_pvc
     pvname = "local0001"
     storage_class_name = "k8s-deploy-test"
 
-    assert_deploy_success(deploy_fixtures("pvc", subset: ["wait_for_first_consumer_storage_class.yml"]))
+    assert_deploy_success(deploy_global_fixtures_non_namespaced("pvc",
```

Comment:
This isn't safe to do in the parallel tests

Reply:
Moved the 3 unsafe tests to the serial suite.